pytket/extensions/qiskit/backends/ibmq_emulator.py

```diff
@@ -14,26 +14,21 @@
 
 from collections import Counter
 from typing import (
-    Dict,
     Optional,
-    List,
     Sequence,
-    Tuple,
-    Union,
+    Any,
 )
 
 from qiskit_aer.noise.noise_model import NoiseModel  # type: ignore
 from qiskit_ibm_runtime import QiskitRuntimeService  # type: ignore
 
-from pytket.backends import (
-    Backend,
-    ResultHandle,
-    CircuitStatus,
-)
+from pytket.backends.backend import Backend
+from pytket.backends.status import CircuitStatus
+
 from pytket.backends.backendinfo import BackendInfo
 from pytket.backends.backendresult import BackendResult
-from pytket.backends.resulthandle import _ResultIdTuple
-from pytket.circuit import Circuit
+from pytket.backends.resulthandle import _ResultIdTuple, ResultHandle
+from pytket._tket.circuit import Circuit
```

Comment:
Not sure why this change is needed?

Reply:
This just stops the typechecker in my IDE from complaining. Can change it back if preferred.

Reply:
OK, I think it's better to revert it; we shouldn't need to use `_tket` in the extensions code I think.

Reply:
No problem, done in [ca9e1a8](https://github.com/CQCL/pytket-qiskit/pull/383/commits/ca9e1a86f0b6600f1e2b8f3ca35a8d0c663f23bc)